lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -1243,7 +1245,7 @@ collectStreams (nOut0, nErr0) p = do
         hSetBuffering o LineBuffering
         hSetBuffering e LineBuffering
         let io = race
-                (threadDelay (65 * oneSecond))
+                (threadDelay (30 * oneSecond))
```

Comment:
Not totally sure on this though. Maybe the default `waitForConnection` should have a chance to timeout.

Reply:
Not sure this has anything to do with the CI failures :thinking:

Reply:
I was lead to believe that, but doesn't seem like that now, no.